app/Models/V2/Projects/Project.php

```diff
@@ -181,8 +172,6 @@ class Project extends Model implements MediaModel, AuditableContract, EntityMode
         'restoration_strategy' => 'array',
         'sdgs_impacted' => 'array',
         'answers' => 'array',
-        'detailed_intervention_types' => 'array',
-        'states' => 'array',
```

Comment:
This removal and the one above is reverting some recent work from staging. Please update your branch to avoid that.

Reply:
done merge from staging